cpp/src/Slice/Parser.cpp

```diff
@@ -448,45 +448,46 @@ namespace
 optional<DocComment>
 Slice::DocComment::parseFrom(const ContainedPtr& p, DocLinkFormatter linkFormatter, bool stripMarkup, bool escapeXml)
```

Comment:
All I did was swap the order, and move the link-fixing into a `for` loop.
Since now it works over a `vector<string> lines` instead of a `string rawComment`.

Since `@link` cannot span multiple lines, this is fine.